examples/consistency_distillation/train_lcm_distill_lora_sd_wds.py

```diff
@@ -1388,6 +1433,8 @@ def compute_embeddings(prompt_batch, proportion_empty_prompts, text_encoder, tok
 
             # Checks if the accelerator has performed an optimization step behind the scenes
             if accelerator.sync_gradients:
+                # 12. If using a target model, update its parameters via EMA.
+                update_ema(target_unet.parameters(), unet.parameters(), args.ema_decay)
```

Comment:
Are you sure this works? I had errors with the LoRA parameters.

Reply:
I have a workaround in the issue

Reply:
both are lora weights, so they should work?

Reply:
I haven't tested this fully yet, it's possible that this runs into the errors mentioned in https://github.com/huggingface/diffusers/issues/6505#issuecomment-1886855011

Reply:
I believe the current implementation is correctly updating the LoRA parameters.